src/control/lib/control/acl.go

```diff
@@ -19,13 +19,36 @@ import (
 
 // AccessControlList is a structure for the access control list.
 type AccessControlList struct {
-	Entries    []string `json:"ACL"`       // Access Control Entries in short string format
-	Owner      string   `json:"OwnerUser"` // User that owns the resource
-	OwnerGroup string   // Group that owns the resource
+	Entries    []string `json:"entries"`     // Access Control Entries in short string format
+	Owner      string   `json:"owner_user"`  // User that owns the resource
+	OwnerGroup string   `json:"owner_group"` // Group that owns the resource
 }
 
-func (acl *AccessControlList) MarshalJSON() ([]byte, error) {
-	return json.Marshal(acl.Entries)
+func (acl *AccessControlList) UnmarshalJSON(data []byte) error {
+	type fromJSON AccessControlList
```

Comment:
It might be worth adding a comment here to explain why this is needed. Could also create a TODO ticket for fixing the protobuf side of things so that this isn't necessary.